src/main/java/com/snowflake/kafka/connector/records/SnowflakeAvroConverter.java

```diff
@@ -43,9 +44,14 @@
   // By default, we don't break when schema registry is not found
   private boolean breakOnSchemaRegistryError = false;
 
+  public static final String READER_SCHEMA = "reader.schema";
```

Comment:
Let's club this together with other public static.

Reply:
Also, doc comments please.

Reply:
Ok!